templates/default/anaconda_env.sh.erb

```diff
@@ -17,6 +17,7 @@ ARG=$4
 OFFLINE=$5
 HADOOP_DIR=$6
 INSTALL_JUPYTER=$7
+YML_FILE_PATH=$8
```

Comment:
@robzor92 can you make sure you sanitize the new input like here: https://github.com/logicalclocks/kagent-chef/pull/110 ?